packages/build-utils/src/prerender.ts

```diff
@@ -62,5 +65,12 @@ export class Prerender {
       );
     }
     this.fallback = fallback;
+
+    if (allowQuery && !Array.isArray(allowQuery)) {
+      throw new Error(
+        'The `allowQuery` argument for `Prerender` needs to be a string Array.'
```

Comment:
Should we do `allowQuery.some(x => typeof x !== 'string')` to make sure it's really all strings?

Reply:
Good point, done!